receiver/kubeletstatsreceiver/internal/kubelet/metadata.go

```diff
@@ -62,12 +62,15 @@ type resources struct {
 	memoryLimit   int64
 }
 
-type NodeCapacity struct {
+type NodeInfo struct {
```

Comment:
Can you make the rename as a first PR? It makes reviewing difficult.

Reply:
done https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39281